app/src/test/java/com/github/sdpteam15/polyevents/database/objects/UserDatabaseTest.kt

```diff
@@ -303,4 +312,44 @@ class UserDatabaseTest {
         assertEquals(PROFILE_COLLECTION, get.collection)
         assertEquals(ProfileAdapter, get.adapter)
     }
+
+    @Test
+    fun getListUsersWorksProperly() {
+        val mockedDatabase= mock(FirebaseFirestore::class.java)
+        FirestoreDatabaseProvider.firestore =mockedDatabase
+        val mockedTask = mock(Task::class.java) as Task<QuerySnapshot>
+        val mockedCollection = mock(CollectionReference::class.java)
+        val mockedDocument = mock(QuerySnapshot::class.java)
+
+        val userToBeAdded = mutableListOf<UserEntity>()
+        val testUser = ObservableList<UserEntity>()
+        userToBeAdded.add(UserEntity("uid1","username1","name1"))
+        userToBeAdded.add(UserEntity("uid2","username2","name2"))
+        userToBeAdded.add(UserEntity("uid3","username3","name3"))
+        userToBeAdded.add(UserEntity("uid4","username4","name4"))
+        userToBeAdded.add(UserEntity("uid5","username5","name5"))
+        userToBeAdded.add(UserEntity("uid6","username6","name6"))
+        userToBeAdded.add(UserEntity("uid7","username7","name7"))
+        userToBeAdded.add(UserEntity("uid8","username8","name8"))
+        userToBeAdded.add(UserEntity("uid9","username9","name9"))
```

Comment:
Do we need that many test users? ^^

Reply:
Why not, I mean I push Ctrl+D too many times xD

Reply:
Btw, I removed this test